packages/babel-plugin-fbtee/src/__tests__/fbtPronoun-test.tsx

```diff
@@ -1,57 +1,26 @@
-import { PluginOptions } from '@babel/core';
 import { describe, expect, it } from '@jest/globals';
 import {
-  assertSourceAstEqual,
-  payload,
+  jsCodeFbtCallSerializer,
+  snapshotTransform,
   transform,
   withFbtImportStatement,
 } from './FbtTestUtil.tsx';
 
-function runTest(
-  data: { input: string; output: string },
-  extra?: PluginOptions,
-) {
-  const expected = data.output;
-  const actual = transform(data.input, extra);
-  assertSourceAstEqual(expected, actual);
-}
+expect.addSnapshotSerializer(jsCodeFbtCallSerializer);
 
 describe('fbt pronoun support', () => {
   it('"capitalize" option accepts boolean literal true', () => {
-    runTest({
-      input: withFbtImportStatement(
-        `var x = fbt(
+    expect(
+      snapshotTransform(
+        withFbtImportStatement(
+          `var x = fbt(
             fbt.pronoun('possessive', gender, {capitalize: true}) +
               ' birthday is today.',
             'Capitalized possessive pronoun',
           );`,
+        ),
       ),
-
-      output: withFbtImportStatement(
-        `var x = fbt._(
-          ${payload({
-            jsfbt: {
-              m: [null],
-              t: {
-                '*': {
-                  desc: 'Capitalized possessive pronoun',
-                  text: 'Their birthday is today.',
-                },
-                1: {
-                  desc: 'Capitalized possessive pronoun',
-                  text: 'Her birthday is today.',
-                },
-                2: {
-                  desc: 'Capitalized possessive pronoun',
-                  text: 'His birthday is today.',
-                },
-              },
-            },
-          })},
-          [fbt._pronoun(1, gender)],
-        );`,
-      ),
-    });
+    ).toMatchSnapshot();
```

Comment:
Could you use inline snapshot tests instead though? It's much easier to review "userland" fbt usage vs the expected "internal" fbt._() output.